static/scss/dashboard.scss

```diff
@@ -188,16 +188,18 @@ c.validation-wrapper {
     width: 100%;
     position: relative;
     overflow: visible;
-    .add-employment-button {
+    margin-bottom: 20px;
+
+    .profile-add-button {
       left: 40%;
     }
-    .add-employment-button.no-entries {
+    .profile-add-button.no-entries {
       top: 68px;
     }
-    .add-employment-button.one-entry {
+    .profile-add-button.one-entry {
       top: 38px;
     }
-    .add-employment-button.more-entries {
+    .profile-add-button.more-entries {
       top: 28px;
```

Comment:
re: the comment above about spacing, I think we may need to either have different spacing for the education and professional tabs, or do some tweaking to what the education tab currently looks like to ensure that the same `top: XXpx;` style will put the button in the right place (currently it looks like the fonts are a little bigger on the education tab, and that might be causing the issue).